inc/config.class.php

```diff
@@ -242,14 +247,17 @@ public function getFromDBByFunctionAndClassname($function, $classname)
      **/
     public static function addFirstconfigLink()
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
+
         $buttons = [];
         $title   = '';
 
         if (Session::haveRight('config', READ)) {
             $buttons['config.php?new=1'] = __('Initialize graphics configuration', 'mreporting');
         }
         Html::displayTitle(
-            Plugin::getWebDir('mreporting') . '/pics/config2.png',
+            $CFG_GLPI['root_doc'] . '/plugins/mreporting/pics/config2.png',
```

Comment:
Isn't “public/” missing?
```suggestion
            $CFG_GLPI['root_doc'] . '/plugins/mreporting/public/pics/config2.png',
```